### PR TITLE
ROX-24458: Move app interface CI to podman

### PR DESCRIPTION
**Pull request by kurlov**
## Description

Migrate app interface CI script to podman. Also simplify build and push script a little

## Checklist (Definition of Done)

- [ ] Unit and integration tests added
- [ ] Added test description under `Test manual`
- [ ] Documentation added if necessary (i.e. changes to dev setup, test execution, ...)
- [ ] CI and all relevant tests are passing
- [ ] Add the ticket number to the PR title if available, i.e. `ROX-12345: ...`
- [ ] Discussed security and business related topics privately. Will move any security and business related topics that arise to private communication channel.
- [ ] Add secret to app-interface Vault or Secrets Manager if necessary
- [ ] RDS changes were e2e tested [manually](../docs/development/howto-e2e-test-rds.md)
- [ ] Check AWS limits are reasonable for changes provisioning new resources
- [ ] (If applicable) Changes to the dp-terraform Helm values have been reflected in the addon on integration environment

### CODE REVIEW

File: build_push_app_interface.sh [hunk 1]
```diff
@@ -36,69 +36,36 @@ IMAGE_REPOSITORY="${QUAY_IMAGE_REPOSITORY:-app-sre/acs-fleet-manager}"
...
-  QUAY_USER="${QUAY_USER}" \
-  QUAY_TOKEN="${QUAY_TOKEN}" \
   external_image_registry="quay.io" \
-  internal_image_registry="quay.io" \
-  emailsender_image_repository="${EMAILSENDER_IMAGE_REPOSITORY}" \
-  docker/login/fleet-manager \
-  image/push/emailsender
+  image_repository="${EMAILSENDER_IMAGE_REPOSITORY}" \
+  TAG="${VERSION}" \
+  push/app-interface/fleet-manager
```
Comment (ebensh) [inline build_push_app_interface.sh:65]: push/app-interface/email-sender

File: Makefile [hunk 2]
```diff
@@ -577,6 +577,20 @@ image/push/probe: image/build/probe
 	@echo "Image was pushed as $(IMAGE_REF)."
 .PHONY: image/push/probe
 
+build/app-interface/fleet-manager:
```
Comment (ebensh) [inline Makefile:580]: Do we need the .PHONY tag on these build targets

File: build_push_app_interface.sh (hunk 1 above)
Comment (ebensh) [inline build_push_app_interface.sh:65]: s/fleet-manager/emailsender ?

**Review by ebensh — COMMENTED** [review]
Tested?

**Comment by kurlov** [comment]
> Tested?
It is a little tricky to test locally. App interface CI is broken right now so I will not make things worse

**Review by ebensh — APPROVED** [review]

**Pull request merged**